### ISSUE TITLE
CPU Processing Error with MLP?

### ISSUE THREAD

**Issue by Sakari-Woods** [Sakari-Woods/PythonStockPredictor#1]
Received error stating that Tensorflow does not support the current CPU, potential cause due to use of deprecated command. 

Line 148: loss = tf.reduce_mean(tf.nn.softmax_cross_entropy_with_logits(logits=predict,labels=Y))

Issue with tf.nn.softmax_cross_entropy_with_logits
